XamlControlsGallery/Common/VariedImageSizeLayout.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+using VirtualizingLayout = Microsoft.UI.Xaml.Controls.VirtualizingLayout;
+using VirtualizingLayoutContext = Microsoft.UI.Xaml.Controls.VirtualizingLayoutContext;
+
+namespace AppUIBasics.Common
+{
+    public class VariedImageSizeLayout : VirtualizingLayout
+    {
+        public VariedImageSizeLayout()
+        {
+            Width = 150.0;
+        }
+
+        public double Width { get; set; }
```

Comment:
you might be able to intialize to 150 here by saying = 150; and remove the one in the ctor.